deployment/terraform/keycloak.go

```diff
@@ -356,7 +356,7 @@ func (t *Terraform) setupKeycloakAppConfig(sshc *ssh.Client, cfg *model.Config)
 		return fmt.Errorf("error uploading saml-idp.crt: %s - %w", out, err)
 	}
 
-	keycloakUrl := keycloakScheme + "://" + t.output.KeycloakServer.PublicDNS + ":8080"
+	keycloakUrl := keycloakScheme + "://" + t.output.KeycloakServer.PrivateDNS + ":8080"
```

Comment:
This has to be always private?

Reply:
This is the URL to be used from Mattermost to the Keycloak server, so it can be always private since the servers should reside in the same network.